test/jpypetest/test_conversion.py

```diff
@@ -107,9 +107,10 @@ def testCharConversion(self):
         self.assertEqual(JChar._canConvertToJava(bytes([1])), "implicit")
         self.assertEqual(JChar._canConvertToJava(bytes([1, 1])), "none")
 
-    def testCharOverflow(self):
-        with self.assertRaises(OverflowError):
-            JChar(1000000)
+     # This test is wrong 'char q = (char) 1000000;' works in Java
```

Comment:
If a java char is 16 bit long, the cast would overflow. So by "works" you mean there is no exception thrown?

Reply:
Yes.  The test was looking for an exception but by the Java standard there should not be one.   Our implementation is that explicit cases should not throw, but implicit conversions should.   This test was therefore testing for an incorrect behavior.